Assets/Scripts/ViconNexusUnityStream/CustomHWDScript.cs

```diff
@@ -54,13 +54,13 @@ private void OnValidate()
         {
             if (Application.isPlaying)
             {
-                rotFilter = new OneEuroFilter<Quaternion>(90, rotFilterMinCutoff, rotFilterBeta);
-                posFilter = new OneEuroFilter<Vector3>(90, posFilterMinCutoff, posFilterBeta);
+                rotFilter = new(90, rotFilterMinCutoff, rotFilterBeta);
+                posFilter = new(90, posFilterMinCutoff, posFilterBeta);
```

Comment:
Since the variables are not defined here, I would fully qualify the object creation instead of using the short form.